### DISCUSSION TITLE
How to change the background color of PCManFM-Qt?

### DISCUSSION THREAD

**Discussion by frank038** [lxqt/lxqt#2071] [Q&A]
I'd like to change the background color of PCManFM-Qt but at the moment I haven't found any tips about that. How can I do?

**Comment by tsujan** [comment]
With Fusion, Breeze,..., you need to change the palette. See `LXQt Appearance Configuration → Widget Style → Qt palette` if you have a recent version of LXQt.

With Qt styles that don't follow the palette (like Kvantum, gtk2,...), you need to change their theme

**Reply by frank038** [reply]
Thank you for your answer.
My LxQt version is still 0.14.1.
Any chance with the option -stylesheet?

**Reply by tsujan** [reply]
> My LxQt version is still 0.14.1.

It's very old and not recommended.

> Any chance with the option -stylesheet?

No. All Qt apps, except for some special apps (like lxqt-panel), are styled by the active widget style and its palette. That's a part of what is called "platform integration".

**Reply by frank038** [reply]
Ok. Thank you very much.
